huaweicloud/services/modelarts/resource_huaweicloud_modelarts_notebook.go

```diff
@@ -400,12 +396,12 @@ func buildEndpointsParamter(d *schema.ResourceData) []notebook.EndpointsReq {
 	return nil
 }
 
-func setVolumeToState(d *schema.ResourceData, config notebook.VolumeRes) error {
+func setVolumeToState(d *schema.ResourceData, cfg notebook.VolumeRes) error {
```

Comment:
how about use **volume**? I think `config` and `cfg` are not suitable

Reply:
ok,done